src/tasks/minions/minigames/nightmareZoneActivity.ts

```diff
@@ -0,0 +1,57 @@
+import { resolveAttackStyles } from '../../../lib/minions/functions';
+import { incrementMinigameScore } from '../../../lib/settings/settings';
+import { SkillsEnum } from '../../../lib/skilling/types';
+import { handleTripFinish } from '../../../lib/util/handleTripFinish';
+import { NightmareZoneActivityTaskOptions } from './../../../lib/types/minions';
+
+export const nightmareZoneTask: MinionTask = {
+	type: 'NightmareZone',
+	async run(data: NightmareZoneActivityTaskOptions) {
+		const { quantity, userID, channelID, duration, strategy } = data;
+		const user = await mUserFetch(userID);
+
+		// No monster ID so use -1 as placeholder
+		const [, , attackStyles] = resolveAttackStyles(user, { monsterID: -1 });
```

Comment:
This is a bad thing to do. I'd much rather you refactored `resolveAttackStyles` so the monsterID is optional.